src/LinAlg/hiopVectorRajaImpl.hpp

```diff
@@ -130,28 +132,27 @@ hiopVectorRajaPar::hiopVectorRajaPar(
   n_local_ = glob_iu_ - glob_il_;
 
 #ifndef HIOP_USE_GPU
-  mem_space_ = "HOST"; // If no GPU support, fall back to host!
+  assert(mem_space_ == "HOST"); 
```

Comment:
#577 applies here - perhaps in the meantime we should give an error message with the assert.